testgrid/config.yaml

```diff
@@ -4612,7 +4614,10 @@ dashboards:
   dashboard_tab:
   - name: gce-new-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gce-new-master-upgrade-master
-    description: Upgrade master only, in gce(gci), from 1.10 to master
+    description: Upgrade master only, in gce(gci), from 1.10 to master, run non-parallel-safe tests only
```

Comment:
should this be 1.11 now?

Reply:
I _think_ so.  Maybe we should change these to say `from stable1` so that we don't need to change it, or maybe `from stable1 (1.11)` to make future updates easier?